config/application.yml.default

```diff
@@ -157,6 +157,9 @@ idv_available: true
 idv_contact_phone_number: (844) 555-5555
 idv_max_attempts: 5
 idv_min_age_years: 13
+idv_resolution_alternate_vendor: mock
+idv_resolution_alternate_vendor_percent: 0
+idv_resolution_default_vendor: mock
```

Comment:
Yeah that's a good suggestion, but in this case I'm match patterns in place elsewhere in our config (not saying those are correct, but I think consistency is good). In practice right now you won't ever use 3 vendors, so this feels like a problem we can figure out how to solve when we get there.

Some of this is also speculative--I'm assuming that our strategy for determining which vendor to send to will be random chance, but it could also be informed by what we learn during the shadow mode pilot.